projects/packages/forms/src/contact-form/css/grunion.scss

```diff
@@ -437,7 +437,7 @@ that needs to mimic the input element styles */
 }
 
 .wp-block-jetpack-contact-form .grunion-field-width-auto-wrap {
-	flex: 1 1 auto;
+	flex: 1 1 0;
```

Comment:
The main issue identified in this PR is working well for me - confirming that when there are two side-by-side auto-width fields, validation errors don't snap them to 100% width. But it's also worth noting I could not duplicate that issue on trunk either. 

I did find this separate issue (exists on trunk too) that @CGastrell  is looking at. I think a fix will be included here. Will re-test when that's added. 

<img width="711" height="402" alt="auto-width-form-error" src="https://github.com/user-attachments/assets/909cfea7-e070-465d-ba73-c063293a2cce" />

Reply:
Thanks!